src/saveload/compat/settings_sl_compat.h

```diff
@@ -95,13 +95,9 @@ const SaveLoadCompat _settings_sl_compat[] = {
 	SLC_VAR("vehicle.smoke_amount"),
 	SLC_NULL(1, SL_MIN_VERSION, SLV_159),
 	SLC_VAR("pf.roadveh_queue"),
-	SLC_VAR("pf.new_pathfinding_all"),
-	SLC_VAR("pf.yapf.ship_use_yapf"),
-	SLC_VAR("pf.yapf.road_use_yapf"),
-	SLC_VAR("pf.yapf.rail_use_yapf"),
-	SLC_VAR("pf.pathfinder_for_trains"),
-	SLC_VAR("pf.pathfinder_for_roadvehs"),
-	SLC_VAR("pf.pathfinder_for_ships"),
+	SLC_NULL(1, SL_MIN_VERSION, SLV_87),
+	SLC_NULL(3, SLV_28, SLV_87),
+	SLC_NULL(3, SLV_87, SLV_TABLE_CHUNKS),
```

Comment:
Why did you choose this old version instead of creating `SLV_REMOVE_NPF`?

Reply:
Probably because the old way of storing went away with the introduction of "table chunks" and as such the more descriptive storing of the settings (with name) in the saves. Normally the versioning is configured to the settings ini files, and global variables are allocated to temporarily save the information in when it's needed for afterload purposes. In these cases they can completely be removed from the settings ini file, but for loading old games we still need to know which bytes of the pre-table-chunks saves we need to skip.

This has been done this way for other settings that got removed.